docs/connect.md

```diff
@@ -128,14 +128,14 @@ const inviteLinkCid = await cx.shareWith(myShareToken)
 The resulting CID looks something like this: `bafkreifzpfmjjmkrqwtzazzj46l6m4sa4umhyd5fv7hvjauyz2eftcvbda` -- this material is safe to publish because it can only be utilized by someone who has the private key the share token corresponds with. Send this `inviteLinkCid` to the person who's share token you used to initiate the share. They can then call:
 
 ```js
-const { database, connection } = await cx.joinShared(inviteLinkCid)
+const { ledger, connection } = await cx.joinShared(inviteLinkCid)
```

Comment:
this is old

Reply:
we should. leave it in for now and then do a cleanup

Reply:
opened #12